etc/evince.profile

```diff
@@ -28,6 +28,7 @@ protocol unix
 seccomp
 shell none
 tracelog
+net none
```

Comment:
commented out, breaks AppArmor on Ubuntu systems

Reply:
could  you  add a comment?.

just like.

>#please uncomment if you dont have apparmor *archlinux*